common/djangoapps/third_party_auth/migrations/0025_auto_20200225_1646.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-02-25 16:46
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('third_party_auth', '0024_fix_edit_disallowed'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='providerapipermissions',
+            name='client',
+        ),
+        migrations.DeleteModel(
+            name='ProviderApiPermissions',
```

Comment:
@feanil @dianakhuang: What is the correct way to hold off on this migration until post-Juniper and tell Django not to be concerned with it?